pages/common/column.md

```diff
@@ -0,0 +1,19 @@
+# column
+
+> Format standard or file input into multiple columns, rows are filled before columns.
+
+- Output is formatted for a 10 columns wide display:
+
+`printf "header1 header2\nbar foo\n" | column -c {{10}}`
+
+- Specify characters to use to delimit columns for the -t option (i.e. "," for csv), default is whitespace:
+
+`printf "header1,header2\nbar,foo\n" | column -s{{,}}`
+
+- Determine the number of columns and auto-align the output in a tabular format:
```

Comment:
"Split columns automatically" is fine, but the default separator is whitespace or the character passed with -s {{}}.

How does this sounds?

`- Split columns automatically; default separator is whitespace or passed by -s :`